frontend/talentsearch/src/js/lang/fr.json

```diff
@@ -1636,5 +1636,283 @@
   "GIPciq": {
     "defaultMessage": "<strong>Remarque :</strong> Si vous sélectionnez plus d'un groupe visé par l'équité en matière d'emploi, TOUS les candidats qui se sont déclarés membres de N'IMPORTE LEQUEL des groupes visés par l'équité en matière d'emploi sélectionnés seront recommandés. Si vous avez des exigences plus détaillées concernant l'équité en matière d'emploi, veuillez nous en faire part dans la section des commentaires du formulaire de soumission.",
     "description": "Context for employment equity filter in search form."
+  },
+  "+cy2GO": {
+    "defaultMessage": "Trouvez des candidats ayant les bonnes compétences pour le poste. Utilisez les onglets suivants pour trouver les compétences nécessaires à l'emploi et sélectionnez-les pour les utiliser comme filtres pour trouver les candidats correspondants.",
+    "description": "Describing how to use the skill filters on search page, paragraph one."
+  },
+  "+pWIcS": {
+    "defaultMessage": "Compétences techniques",
+    "description": "Button text for the technical skills tab on skills filter"
+  },
+  "/Gu4zR": {
+    "defaultMessage": "Expérience postsecondaire de deux ans",
+    "description": "Title for pool applicant education requirements"
+  },
+  "0FjYi+": {
+    "defaultMessage": "Pour être admis dans ce processus, vous devrez faire preuve de capacités dans ces compétences au cours du processus d'évaluation",
+    "description": "Explanation of a pools required transferrable skills"
+  },
+  "0I8W8B": {
+    "defaultMessage": "Compétences transférables",
+    "description": "Title for transferrable skills on a pool advertisement"
+  },
+  "18dDgn": {
+    "defaultMessage": "À propos de ce processus",
+    "description": "Title for the about section of a pool advertisement"
+  },
+  "2K8q04": {
+    "defaultMessage": "Avez-vous besoin de mesures d’adaptation, ou avez-vous des questions sur ce processus?",
+    "description": "Opening sentence asking if accommodations are needed"
+  },
+  "5IfAK9": {
+    "defaultMessage": "Demande de candidats pour le bassin : <span>{candidateCount, plural, zero {aucun candidat} one {1 candidat} other {{candidateCount} candidats estimés}}</span>",
+    "description": "Total estimated candidates message in summary of filters"
+  },
+  "6OJhwM": {
+    "defaultMessage": "Trouver et sélectionner les compétences",
+    "description": "Subtitle for the skills filter on the search form."
+  },
+  "7GXovm": {
+    "defaultMessage": "<strong>Courriel</strong> : <accommodationEmail>fames@acanteipsum.ca</accommodationEmail>"
```

Comment:
Is it possible to add a description to the English string so that one can find the English string if necessary?